docs/resource/cloud.md

```diff
@@ -39,50 +39,6 @@ The following well-known definitions MUST be used if you set this attribute and
   `/subscriptions/<SUBSCIPTION_GUID>/resourceGroups/<RG>/providers/Microsoft.Web/sites/<FUNCAPP>/functions/<FUNC>`.
   This means that a span attribute MUST be used, as an Azure function app can host multiple functions that would usually share
   a TracerProvider.
-
-`cloud.platform` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
```

Comment:
We lost now the table with the values. You need to use the `full` syntax, like in here https://github.com/open-telemetry/semantic-conventions/blob/main/docs/rpc/rpc-spans.md?plain=1#L85.

Reply:
Thank you so much for pointing this out!! It should be fixed now.